app/src/main/java/com/apps/adrcotfas/goodtime/BL/NotificationHelper.java

```diff
@@ -115,17 +114,17 @@ private void initChannels() {
             if (currentSession.getTimerState().getValue() == TimerState.PAUSED) {
                 builder.addAction(buildStopAction(this))
                         .addAction(buildResumeAction(this))
-                        .setContentTitle("Work session is paused")
-                        .setContentText("Continue?");
+                        .setContentTitle(getResources().getString(R.string.work_session_is_paused))
```

Comment:
These should be renamed to something more explicit like:
R.string.action_paused_title
R.string.action_continue